front/app/containers/Admin/projects/project/participationContext/components/ParticipationMethodPicker.tsx

```diff
@@ -128,8 +147,10 @@ export const ParticipationMethodPicker = ({
               value="native_survey"
               name="participationmethod"
               id={'participationmethod-native_survey'}
+              disabled={isExistingProjectOrPhase ? true : false}
```

Comment:
Since `isExistingProjectOrPhase` is a boolean, we could just add that here and remove the ternary operator